### DISCUSSION TITLE
Bug Hunting 🐛🔍

### DISCUSSION THREAD

**Discussion by Abdeltif1** [SOEN345-WINTER2024/discussion#13] [General]
.

**Comment by stan6** [comment]
Hi, @Abdeltif1 ,are you TAs for Prof. Ali's section? A reminder that this is discussion for my section.Their section should be under the https://github.com/orgs/SOEN345-Winter24OtherSection/discussions. Please post everything there to avoid confusion.

**Comment by stan6** [comment]
Closing to avoid confusion
